src/api/config/routes/webui_routes.rb

```diff
@@ -292,8 +292,7 @@
 
       resources :subscriptions, only: [:index], controller: 'webui/users/subscriptions', as: :my_subscriptions do
         collection do
-          put 'update'
-          get 'update'
+          put 'update', as: :update
         end
       end
```

Comment:
```suggestion
           resources :subscriptions, only: [], controller: 'webui/users/subscriptions' do
        collection do
          get '/', action: :index, as: :my
        end
        member do
          put 'update'
        end
      end
```

leads to
```
Prefix               Verb     URI Pattern                               Controller#Action
my_subscriptions     GET      /my/subscriptions(.:format)               webui/users/subscriptions#index
subscription         PUT      /my/subscriptions/:id/update(.:format)    webui/users/subscriptions#update
```

Reply:
`EventSubscriptionForm::update` oh my. Why straightforward if you can also do it complicated... Nevermind me.